web-ui/src/api/notifications.js

```diff
@@ -46,3 +47,34 @@ export const sendEmail = async (subject, content, html, recipients, cookie) => {
     }
   });
 };
+
+export const emailPDLAssignment = async (member, cookie) => {
+  if (member.pdlId && member.lastName && member.firstName && member.workEmail) {
+    let res = await getMember(member.pdlId)
+    let pdl =
+        res.payload && res.payload.data && !res.error
+            ? res.payload.data
+            : null;
+    if (pdl && pdl.workEmail) {
+      await sendEmail("You have been assigned as the PDL of " + member.firstName + " " + member.lastName,
+          member.firstName + " " + member.lastName +
+          " will now report to you as their PDL. Please engage with them: " + member.workEmail,
+          false, [pdl.workEmail], cookie)
+    }
+  }
+}
+export const emailSupervisorAssignment = async (member, cookie) => {
+  if (member.supervisorid && member.lastName && member.firstName && member.workEmail) {
```

Comment:
I wonder if it's okay for this to silently do nothing when these criteria are not met. Maybe it should write an error message to the console.

Reply:
Added.